scipy/interpolate/fitpack2.py

```diff
@@ -588,9 +602,9 @@ class InterpolatedUnivariateSpline(UnivariateSpline):
     UnivariateSpline : Superclass -- allows knots to be selected by a
         smoothing condition
     LSQUnivariateSpline : spline for which knots are user-selected
+    BivariateSpline : A similar class for two-dimensional spline interpolation
```

Comment:
Nit: `UnivariateSpline` is meant to be instantiated while `BivariateSpline` is a base class. So maybe, just maybe, link to LSQBivariateSpline and SmoothBivariateSpline instead.

Reply:
I think this is not needed in SeeAlso because it is one of the UnivariateSpline function. So I removed it.